lib/cli/kit/command_registry.rb

```diff
@@ -74,6 +82,20 @@ def command_names
       def exist?(name)
         !resolve_command(name).first.nil?
       end
+
+      private
+
+      def snake_case(camel_case, seperator = "_")
+        camel_case.to_s
```

Comment:
We've tended towards leaving a command providing an example transformation, e.g.

```
camel_case.to_s                                         # MyCoolThing::MyCoolModule
.gsub(/::/, '/')                                        # MyCoolThing/MyCoolModule
.gsub(/([A-Z]+)([A-Z][a-z])/, "\\1#{seperator}\\2")     # I'm not sure what's happening here actually
.gsub(/([a-z\d])([A-Z])/, "\\1#{seperator}\\2")         # I'm not sure what's happening here actually
.downcase
```

Reply:
Updated.